python/mxnet/symbol.py

```diff
@@ -1048,6 +1048,12 @@ def creator(*args, **kwargs):
         for k, v in kwargs.items():
             if isinstance(v, Symbol):
                 symbol_kwargs[k] = v
+            elif isinstance(v, list):
```

Comment:
you don't need this. if you have a list of input say [x, y, z,]
you can do caffeop(x, y, z, xxx=yyy, ...) or caffeop(*[x, y, z], xxx=yyy, ...)